src/migrations/1666964653220_remove-snapshot-signature-column-in-proposal.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { ColumnDefinitions, MigrationBuilder } from 'node-pg-migrate'
+
+import Model from '../entities/Proposal/model'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.dropColumn(Model.tableName, 'snapshot_signature')
+}
+
+export async function down(pgm: MigrationBuilder): Promise<void> {
+  pgm.addColumn(Model.tableName, {
+    snapshot_signature: {
+      type: 'TEXT',
+      notNull: true,
```

Comment:
if it is not null, shouldn't be a default value?

Reply:
Use the same object as in the create proposals migration https://github.com/decentraland/governance/blob/master/src/migrations/1616763227514_create-proposals.ts#L26.
I think if it is notNull without default value it means you can't create a row without that value. If it has a default value you can avoid passing a value to the row and the default is set (and it is always not null).